examples/megacli.cpp

```diff
@@ -4373,16 +4372,26 @@ void exec_sync(autocomplete::ACState& s)
             }
             else
             {
-                static int syncTag = 2027;
-                SyncConfig syncConfig{syncTag++, s.words[1].s, s.words[1].s, n->nodehandle, s.words[2].s, 0, {}, true, newSyncConfig.getType(),
+                SyncConfig syncConfig{s.words[1].s, s.words[1].s, n->nodehandle, s.words[2].s, 0, {}, true, newSyncConfig.getType(),
                             newSyncConfig.syncDeletions(), newSyncConfig.forceOverwrite()};
-                UnifiedSync* unifiedSync;
-                error e = client->addsync(syncConfig, DEBRISFOLDER, NULL, false, unifiedSync, true);
 
-                if (e)
-                {
-                    cout << "Sync could not be added: " << errorstring(e) << endl;
-                }
+                cout << "Adding sync..." << endl;
+
+                client->addsync(syncConfig, DEBRISFOLDER, NULL, false, false,
+                        [](mega::UnifiedSync* us, const SyncError&, error e){
+                            if (us && us->mSync)
+                            {
+                                cout << "Sync added and running. backupId = " << us->mConfig.getBackupId();
```

Comment:
Note: printing and inputing of backupId is done considering it a integer. I wonder if we should try and use a base64 representation :thinking:  ... Same applies to MEGAcmd, that would require similar changes. What do you think @mattw-mega ?

Reply:
Yes, please. Output should always print handles in B64, since that's the format provided by the API and it helps a lot to identify them in logs.
As for input params, I think it's better to ask for a `MegaHandle`/`handle`, which is the common practice already and more efficient than a B64 string.

Reply:
yes, base64 would be a good way to go for identification in text, agreed